### DISCUSSION TITLE
The current state of GitHub education

### DISCUSSION THREAD

**Discussion by Christian-Schwarz2003** [community/community#60742] [GitHub Education]
### Hi everyone,

A couple months ago GitHub got rid of manual (human) review of applications for students, this was the first bad omen.

Then my academic e-mails were required (even at institutions that don't provide those to students.

I had to contact support and they removed the requirement for my institution after some back and forth (this was April).

Even then I had to try 20+ times to get my application approved until I had the right angle on the document.

End of June I had to reverify my account and the e-mail requirement is back and GitHub support hasn't responded in over 2 weeks.

The current state of GitHub education is absolutely horrible for students that don't have academic e-mails.

### A suggestion...

Don't add impossible requirements, and bring back human reviews!

**Comment by BenjaminDecaillet** [comment]
Hello,
similar issue here. I have a school issued e-mail for students but it is not matching what they request and I cannot re-submit my application.

I cannot reach out to support it seems now about Github Education. 

Suggestions?

![Re-submit_Github](https://github.com/community/community/assets/26813570/642026d6-f3ee-49a2-8a1c-43c42676c179)

**Comment by samus-aran** [comment]
Hi @Christian-Schwarz2003  👋 
We're sorry to hear of your frustration. We are working our way through addressing all the support tickets as fast as possible. 
I can share that your school info has since been updated. Can we please ask you to try now and re-apply. We hope this helps you proceed to re-verify your academic status. 👍
